### DISCUSSION TITLE
Doge coin transaction fee control - Android

### DISCUSSION THREAD

**Discussion by serin-sebastian** [trustwallet/wallet-core#2224] [Q&A]
How can I set the transaction fee for a doge coin transaction. I Am attaching my code sample here. What changes do I need to make?
![transaction fees](https://user-images.githubusercontent.com/94120034/169034641-1b989ac8-1cc4-4064-ab5e-ec6e67e3502e.png)

**Comment by optout21** [comment]
See also: https://github.com/trustwallet/wallet-core/discussions/2222
